src/ImageGallery.jsx

```diff
@@ -490,7 +508,12 @@ export default class ImageGallery extends React.Component {
   }
 
   handleKeyDown = (event) => {
-    if (this.props.disableArrowKeys) {
+    if (this.props.disableArrowKeys // arrow keys are disabled in general
```

Comment:
We want to fix this not disable it.

Reply:
Not sure what you mean, I don't disable it. Can you explain what you mean, please?

Reply:
I guess I'm not understanding the need for `arrowKeysInFullscreenOnly`. The arrow keys should only have one prop that turns it off which is `disableArrowKeys`.

If we want the arrow keys disabled in fullscreen or not fullscreen we should just use `disableArrowKeys`.

Reply:
FTR: solved differently https://github.com/xiaolin/react-image-gallery/pull/446/commits/4368ab2b74c965fc90dc4c35e51e48b20dcb57a6